public/plugin.ts

```diff
@@ -57,9 +58,9 @@ export class WazuhPlugin implements Plugin<WazuhSetup, WazuhStart, WazuhSetupPlu
       id: `wazuh`,
       title: 'Wazuh',
       icon: core.http.basePath.prepend(
-        logosInitialState?.logos?.[SIDEBAR_LOGO] ?
+        logosInitialState?.logos?.[SIDEBAR_LOGO] !== getSettingDefaultValue(SIDEBAR_LOGO) ?
           getAssetURL(logosInitialState?.logos?.[SIDEBAR_LOGO]) :
-          getThemeAssetURL('icon.svg', UI_THEME)),
+          getThemeAssetURL(logosInitialState?.logos?.[SIDEBAR_LOGO], UI_THEME)),
```

Comment:
**nitpick**: This `icon` is the plugin icon, that currently is not displayed on the UI because it only displays the category icon.

Reply:
I will revert this change.